tests/test_verifier.py

```diff
@@ -16,7 +16,7 @@
 
 from unittest import TestCase, skipIf
 
-from django_ses.utils import BounceMessageVerifier, _CERT_CACHE
```

Comment:
This is why I was thinking it's still possible to import `_CERT_CACHE` directly, do you think it's worth adding the `__all__` change to `utils.py` to prevent this?

Reply:
I just did a bit of research on this. From what I've gathered, adding `__all__` wouldn't make a difference here. You could still import it with:

```
 from django_ses.utils import  _CERT_CACHE 
```

The only thing that using `__all__` affects is whether it's imported by this:

```
from django_ses.utils import *
```

But if you begin the object's name with an underscore, as we have here, it's not imported by `*` anyway. So, I don't think adding `__all__`  would even do what we'd hoped.

Details here: https://stackoverflow.com/questions/44834/can-someone-explain-all-in-python